lib/command.js

```diff
@@ -347,7 +348,7 @@ const commands = {
 			statusBar.show("Discarding...", null);
 
 			// discard files
-			return Promise.all([git.clean(root, untrackedFiles), git.checkout(root, trackedFiles)])
+			return Promise.all([git.clean(root, untrackedFiles), git.checkoutFiles(root, trackedFiles)])
```

Comment:
I renamed the existing git.checkout() method to git.checkoutFiles().  git.checkout() now is for checking out a branch.  I thought separate methods was cleaner, but if you prefer we could merge this all back into a single .checkout() method and do some logic on optional arguments.